BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -55,6 +56,11 @@
 @property (nonatomic, readonly, strong) BOXUserMini *user;
 
 /**
+ * The delegate for the BOXContentClient instance (Internally BOXAPIQueueManager)
```

Comment:
Improve the comment here. What is this delegate used for? what does it do? (briefly)